src/flb_engine.c

```diff
@@ -694,6 +695,10 @@ int flb_engine_start(struct flb_config *config)
             flb_sched_timer_cleanup(config->sched);
             flb_upstream_conn_pending_destroy_list(&config->upstreams);
         }
+
+        if (is_error_reporting_enabled()) {
+            flb_info("[engine] error reporting ttl fresh");
```

Comment:
what's the point of this message? Also I don't understand what it means- "error reporting ttl fresh"?

Reply:
Currently we are depend on the fluent bit log event to refresh and remove the expired message inside error reporting file. Just in case some time there is not many logs inside some plugin, add one log in main loop to make sure the refresh will happen at least every 1.5 sec